x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -691,6 +709,12 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                 // Legacy "Alerts as data" used in Security Solution.
                 // Kibana user creates these indices; reads / writes to them.
                 RoleDescriptor.IndicesPrivileges.builder().indices(ReservedRolesStore.ALERTS_LEGACY_INDEX).privileges("all").build(),
+                // Used in Security Solution for value lists.
+                // Kibana user creates these indices; reads / writes to them.
+                RoleDescriptor.IndicesPrivileges.builder().indices(ReservedRolesStore.LISTS_INDEX).privileges("all").build(),
+                // Used in Security Solution for value lists.
+                // Kibana user creates these indices; reads / writes to them.
+                RoleDescriptor.IndicesPrivileges.builder().indices(ReservedRolesStore.LISTS_ITEMS_INDEX).privileges("all").build(),
```

Comment:
The comments for the two code lines are identical. Should they be? If that's true, we can merge the privilege grants, e.g.:
```
RoleDescriptor.IndicesPrivileges.builder().indices(ReservedRolesStore.LISTS_INDEX, ReservedRolesStore.LISTS_ITEMS_INDEX).privileges("all").build(),
```

Reply:
Thanks, I will change it to use that instead as yes, the comments are the same.